workshops/weather_app/README.md

```diff
@@ -0,0 +1,384 @@
+HTML elements and `moment.js`, a Javascript library for managing dates and time.
+
+To link the css file we add
+
+```html
+<link rel="stylesheet" href="style.css">
+```
+under the `head` tag and to link the JS file we add
+
+```html
+<script src="app.js"></script>
+``` 
+under the body tag.
+
+To link external files like `normalize.css` and `moment.js` we use [`CDNJS`](https://cdnjs.com/) which is an open-source CDN service powered 
+by Cloudflare. Cloudflare makes it easier to load library files on our websites.
+
+Add 
+
+```html
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.0/moment.min.js"></script>
+``` 
+under the `head` tag.
+
+Since, we have linked all our files with the `HTML` file now we can start making a card on which we would display the content.
+Personally, I feel cards are a very aesthetic form to display our content. Also, there is a lot of scope of making the cards look better and
+more catchy.
+
+P.S: For more information about cards, refer to `Bootstrap`, an open source `CSS` framework. 
+[Bootstrap Card](https://getbootstrap.com/docs/4.0/components/card/)
+
+Inside our `<body>` we will add `<div>` with class as `container` which makes our main card and for the content inside we can add another
+`<div>` with class as `content`.
+
+Then add a heading using the `h1` tag giving the title to the project.
+
+To input the city from the user we use the `input` tag and declare it with class as `input` and `id` as `input`. 
+
+Mainly `class` is used for passing information in `CSS` and `id` is used for `JS`. 
+Also add a `placeholder` which is a text behind the input label that describes the expected value of input.
+
+```html
+<input id="input" class="input" placeholder="Enter the City Name">
+```
+
+Make sure you close the `div` of `content` because now we have taken the city name from the user and our only task is to display the weather 
+details.
+
+Lastly we have to add `div` with class as `main-weather` to display the weather details.
+
+We will be using `<p>` for every detail we display. Hence we declare the weather details with the following ids
+
+```html
+<p id="date">Date </p>
+<p id="city">City </p>
+<p class="temp" id="temp">Temp </p>
+<p id="min-max">Min and Max Temp </p>
+<p id="weather-type">Sunny </p>
+```
+
+Our `HTML` code is done and now we can move to the `CSS` part. At the end, the `HTML` code will look like:
+
+![html code](https://cloud-ad455y4xg.vercel.app/0screenshot_from_2020-10-09_19-36-08.png)
+
+## 4. Improving the design of skeleton using CSS
+
+PRO TIP: Personally, I feel before we start coding for any project, we should try to make a flowchart containing all the details that we have to include in our project. This makes things clearer and the chances of committing mistakes also reduces.

```

Comment:
This is kind of weird being at step 4. If you want to include this maybe move it earlier in the workshop